chainer/functions/array/vstack.py

```diff
@@ -52,10 +52,11 @@ def vstack(xs):
         xs (list of :class:`~chainer.Variable` or :class:`numpy.ndarray` or \
         :class:`cupy.ndarray`):
             Input variables to be concatenated. The variables must have the
-            same ``ndim``. Also, when the variables have vertical dimension
-            (i.e. :math:`ndim \\geq 1`), the variables must have the same
-            shape, except in the vertical (row wise) dimension.
-
+            same ``ndim``. When the variables have the second axis (i.e.
```

Comment:
So, I misunderstood thant 1-dim array is column vector, but actually it is row vector.
Is it right?